programs/Makefile

```diff
@@ -39,7 +39,7 @@ RELEASE?= r131
 DESTDIR?=
 PREFIX ?= /usr/local
 CFLAGS ?= -O3
-CFLAGS += -std=c99 -Wall -Wextra -Wundef -Wshadow -Wcast-qual -Wcast-align -Wstrict-prototypes -pedantic -DLZ4_VERSION=\"$(RELEASE)\"
+CFLAGS += -DXXH_NAMESPACE=LZ4_ -std=c99 -Wall -Wextra -Wundef -Wshadow -Wcast-qual -Wcast-align -Wstrict-prototypes -pedantic -DLZ4_VERSION=\"$(RELEASE)\"
```

Comment:
`-DXXH_NAMESPACE` is only required to compile xxhash.
Since, in the rest of the file, programs are linked against lz4 library and do not compile xxhash anymore, this define seems unnecessary.